src/wp-includes/class-wp-block.php

```diff
@@ -275,11 +274,18 @@ private function process_block_bindings( $block_content ) {
 
 			// If the value is not null, process the HTML based on the block and the attribute.
 			if ( ! is_null( $source_value ) ) {
-				$modified_block_content = $this->replace_html( $modified_block_content, $attribute_name, $source_value );
+				$computed_attributes[ $attribute_name ] = $source_value;
 			}
 		}
 
-		return $modified_block_content;
+		// If there are computed attributes, update the attributes with the
+		// computed ones.
+		if ( ! empty( $computed_attributes ) ) {
+			// Merge the computed attributes with the original attributes
```

Comment:
Missing end period.

Reply:
@gziolo ☝️

Reply:
I noticed that myself, too. 🙈 I'm working on another commit and I plan to include it. Thank you as always for your attention to detail and checking new commits 🙇🏻

Reply:
Done in https://github.com/WordPress/wordpress-develop/commit/acad00ecabcf44a6c4e552c4fd518ee708fd131f.